### ISSUE TITLE
Provide Docker container

### ISSUE THREAD

**Issue by J0WI** [transmission/transmission#750]
Would be great to have one. :)
Or a dedicated repo for the Dockerfiles where the community can contribute to.

**Comment by jfeilbach** [comment]
https://github.com/linuxserver/docker-transmission

**Comment by J0WI** [comment]
There are some third parties providing one, but upstream support would be great.
See also https://github.com/docker-library/official-images/blob/master/README.md

**Comment by jfeilbach** [comment]
Agreed. However, considering there are 55 pending pull requests going back as far as Jun 17, 2016 it would seem this project is in need of some serious help in order to stay alive, if it isn't already dead. I think getting a Docker container supported is great but less important than keeping the project going.

**Comment by xelra** [comment]
@J0WI linuxserver.io containers are as mainstream as they get. Since they focus on the Docker environment with over 1 billion containers served, in my opinion they're a better choice than first-party containers.

**Comment by PeterDaveHello** [comment]
I'm one of the maintainers of Node.js official Docker image(https://github.com/nodejs/docker-node/) and would like to help if this project would like to have one. Transmission Official would be easier than Docker official. As @jfeilbach mentions, there are some pending and old PRs, I'm not one of the developers of Transmission here, so not sure if this project wants one.

**Comment by ckerr** [comment]
Hey, I just saw this while going through unaddressed issues. There is a [PR for this](https://github.com/transmission/transmission/pull/1142) at the moment that's been sitting open for almost two years.

Basically, the project's two or three maintainers don't have the bandwidth to take this on as an ongoing responsibility. My hope is that all the recent coding activity lately will wake the project back up and attract more contributors, and maybe that's what could happen here.

Transmission hasn't had any releases in ages. We'll be entering in a beta release phase for 4.0.0 sometime in the next month or two, so that will be changing. If @PeterDaveHello is willing to make the images and handle bug reports on it, maybe we could do something like `nodejs/docker-node` and 'bless' it by linking to it from the website and/or from `transmission/transmission`.
